n_serial.c

```diff
@@ -48,7 +48,7 @@ const char *serialNoteTransaction(char *json, char **jsonResponse)
         if (segLen > CARD_REQUEST_SERIAL_SEGMENT_MAX_LEN) {
             segLen = CARD_REQUEST_SERIAL_SEGMENT_MAX_LEN;
         }
-        _SerialTransmit((uint8_t *)&transmitBuf[segOff], segLen, false);
+        _SerialTransmit(&transmitBuf[segOff], segLen, false);

```

Comment:
Based on the type of `transmitBuf` above, this looks like a nice clean-up. :+1:

But just to confirm, the tests running with `-Wall`, `-Wextra`, `-Wpedantic`, `-Werror`, right?

The reason I ask, is because a lot of the casting you see throughout the code was to keep different compilers from issuing warnings about our code. This one seems to be a non-issue, but it made me think about us having our flags enabled.

_I didn't look too hard, but I couldn't immediately find them the `CMakeLists.txt` below._

Reply:
It's in the top-level CMakeLists.txt:

```
add_compile_options(
    -Wall
    -Wextra
    -Wpedantic
    -Werror
)
```

Reply:
I thought I remembered you doing that, but I couldn't find it. :+1: